nikola/nikola.py

```diff
@@ -2633,6 +2579,56 @@ def generic_index_renderer(self, lang, posts, indexes_title, template_name, cont
                 'uptodate': [utils.config_changed(kw, 'nikola.nikola.Nikola.generic_index_renderer')],
             }, kw["filters"])
 
+    def generic_atom_renderer(self, lang, posts, context_source, kw, basename, classification, kind, additional_dependencies=[]):
```

Comment:
Even though it is no problem here, it might be better if you'd use `None` + a bit more logic when you use it. In case someone modifies the code later and forgets about the default argument...

Reply:
[Or would it?](https://github.com/getnikola/nikola/commit/0f067dc31733878ed89538b268891c44d814012c) :wink: